python/src/editor/README.md

```diff
@@ -0,0 +1,35 @@
+# Editor
+
+# Install:
+
+Install python-aiconfig from pip, then set `aiconfig` alias (in shell, .bashrc, .zshrc, etc.)
+
+One-liner:
+`pip install python-aiconfig; alias aiconfig="python -m 'aiconfig.scripts.aiconfig'"`
+
+## Usage
+
+`aiconfig edit  --aiconfig-path=/my/path'`
```

Comment:
just trying to understand if this is the recommended way to run this, or if we'll package this into an actual CLI utility?

Reply:
I think we can wrap this in a shell script and distribute it with pip. I don't know how cross-platform that would be compared to running python directly this way.

Reply:
for ref: https://dzone.com/articles/executable-package-pip-install

Reply:
https://docs.google.com/document/d/1jUsV3Khu_Gs31dIBxh1n2DnF_AbnRPiN9etZpLe1dk8/edit?usp=sharing